assets/src/dashboard/animations/parts/defaultAnimationProps.js

```diff
@@ -46,7 +54,7 @@ export default {
   easingPreset: {
     label: 'Easing Presets',
     type: FIELD_TYPES.DROPDOWN,
-    values: Object.keys(BEZIER),
+    values: ['Use Default', ...Object.keys(BEZIER)],
```

Comment:
I added in "Use Default" as a top level option to our easing preset dropdown so for animation effect/parts like `pulse` and `twirl` we can make sure to use the default easing they already have and not override it with `LINEAR` (which is the default "Easing Presets" value).

Reply:
Awesome. yea I've run into this on other animation effects too. this is better than the localized fix I was doing.

Also think there are a couple animation effects that probably shouldn't have a settable ease anyway right? things like `pulse`, `blink` & `bounce` come to mind there where we're basically creating a custom easing through keyframes